<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Rose’s Thesis…</title>
  <meta name="description" content="Rose’s Thesis…" />
  <meta name="generator" content="bookdown 0.14 and GitBook 2.6.7" />

  <meta property="og:title" content="Rose’s Thesis…" />
  <meta property="og:type" content="book" />
  
  
  
  <meta name="github-repo" content="rosemckeon/honours-project" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Rose’s Thesis…" />
  
  
  

<meta name="author" content="Student number: 2417024" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  

<link rel="next" href="references.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />












<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Rose's Thesis...</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="intro.html"><a href="intro.html#so-why-do-plants-tolerate-genome-doubling-so-well"><i class="fa fa-check"></i><b>1.1</b> So why do plants tolerate genome-doubling so well?</a></li>
<li class="chapter" data-level="1.2" data-path="intro.html"><a href="intro.html#benefits"><i class="fa fa-check"></i><b>1.2</b> Benefits</a><ul>
<li class="chapter" data-level="1.2.1" data-path="intro.html"><a href="intro.html#gigas"><i class="fa fa-check"></i><b>1.2.1</b> Gigas-effects</a></li>
<li class="chapter" data-level="1.2.2" data-path="intro.html"><a href="intro.html#buffering"><i class="fa fa-check"></i><b>1.2.2</b> Genetic buffering</a></li>
<li class="chapter" data-level="1.2.3" data-path="intro.html"><a href="intro.html#vigour"><i class="fa fa-check"></i><b>1.2.3</b> Hybrid Vigour</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="intro.html"><a href="intro.html#costs"><i class="fa fa-check"></i><b>1.3</b> Costs</a><ul>
<li class="chapter" data-level="1.3.1" data-path="intro.html"><a href="intro.html#pollen-swamping"><i class="fa fa-check"></i><b>1.3.1</b> Diploid Pollen-Swamping</a></li>
<li class="chapter" data-level="1.3.2" data-path="intro.html"><a href="intro.html#flowering"><i class="fa fa-check"></i><b>1.3.2</b> Delayed Flowering</a></li>
<li class="chapter" data-level="1.3.3" data-path="intro.html"><a href="intro.html#sterility"><i class="fa fa-check"></i><b>1.3.3</b> Triploid Sterility</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="intro.html"><a href="intro.html#how-do-these-mechanisms-link"><i class="fa fa-check"></i><b>1.4</b> How do these mechanisms link?</a></li>
<li class="chapter" data-level="1.5" data-path="intro.html"><a href="intro.html#aims"><i class="fa fa-check"></i><b>1.5</b> Aims</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Rose’s Thesis…</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="header">
<h1 class="title">Rose’s Thesis…</h1>
<p class="author"><em>Student number: 2417024</em></p>
<p class="date"><em>Last edited: October 24 2019</em></p>
</div>
<div id="intro" class="section level1">
<h1><span class="header-section-number">Chapter 1</span> Introduction</h1>
<blockquote>
<p>Would some images of genome duplication be helpful?</p>
</blockquote>
<p>Whole-genome duplication (polyploidisation) is an extreme mutation which occurs due to errors during cell division, such as non-disjunction; sister chromatids are not separated during anaphase and so all end up in one daughter cell, leaving the other void of genetic material.
When this occurs during gametogenesis it can lead to whole organisms with more than two complete sets of chromosomes (polyploids).
According to <span class="citation">(<span class="citeproc-not-found" data-reference-id="REFNEEDED"><strong>???</strong></span>)</span> the estimated rate of polyploidisation is one order of magnitude higher than that of standard mutation (10<sup>-5</sup>).
In humans whole-genome duplication accounts for around 5% of miscarriages <span class="citation">(<span class="citeproc-not-found" data-reference-id="REFNEEDED"><strong>???</strong></span>)</span>, and only one polyploid mammal has ever been recorded <span class="citation">(<span class="citeproc-not-found" data-reference-id="REFNEEDED"><strong>???</strong></span>)</span>.
But while, in many taxa, this mutation is not well tolerated, the same is not so true of amphibians, fish, fungi and plants.
Polyploidy is exceptionally well tolerated in plants. It’s a ubiquitous feature in the lineages of almost all angiosperms <span class="citation">(Jiao et al. <a href="#ref-Jiao2011">2011</a>)</span>, and it’s been found to have occurred recently (post genera formation) for 35% of all vascular plants <span class="citation">(Wood et al. <a href="#ref-Wood2009">2009</a>)</span>.
Plant surveys of natural populations over the last century further support this, reporting mean polyploid frequencies of 25% <span class="citation">(<span class="citeproc-not-found" data-reference-id="REFNEEDED"><strong>???</strong></span>)</span>.</p>
<div id="so-why-do-plants-tolerate-genome-doubling-so-well" class="section level2">
<h2><span class="header-section-number">1.1</span> So why do plants tolerate genome-doubling so well?</h2>
<p>This process of genome-doubling is thought to be fundamental in the diversification of plant species; having been found to coincide with 15% of angiosperm speciation events, and 31% of ferns <span class="citation">(Wood et al. <a href="#ref-Wood2009">2009</a>)</span>.
Furthermore, <span class="citation">Jiao et al. (<a href="#ref-Jiao2011">2011</a>)</span> showed that whole-genome duplication could be linked with major innovation that lead to diversification of the phyla.
Their phylogenetic analysis brought to light two ancient groups of duplication events (around 319 and 192 MYA).
These first grouping of duplications influenced the diversification of regulatory genes that were integral to seed development and the more recent grouping affected genes that enabled flower development.
Therefore, the authors conclude that these duplications played a major role in the rise of all seed plants, as well as in their later diversification to both gymnosperm and angiosperm phyla.</p>
<p>However, the situation is less clear-cut; according to <span class="citation">Mayrose et al. (<a href="#ref-Mayrose2011">2011</a>)</span> polyploidisation significantly increases speciation rates of diploids, but new polyploid lines don’t further speciate by that same mechanism.
So polyploid speciation rates are smaller in comparison.
Furthermore, polyploid extinction rates are greater than those of diploids.
This Likelihood-based analysis of vascular plants provides clear support for the traditionally popular view that polyploidy most often leads to evolutionary dead ends.
So, what is it then that allows this extreme mutation to succeed and become fixated?
What scenarios and mechanisms combine to allow rare polyploid lines to succeed?
Are the majority of the polyploids we see in natural populations doomed to extinction?
And, can we predict which conditions will lead to their survival?</p>
<p>Despite having been studied for over a century, the factors that drive the success of polyploid establishment in the face of high extinction rates are still unclear.
<span class="citation">Arrigo and Barker (<a href="#ref-Arrigo2012">2012</a>)</span> conclude that polyploids tend to become extinct at the establishment phase due to reproductive disadvantages such as triploid sterility, or limited mate-choice; the latter occurring via diploid pollen-swamping, or delayed flowering.
On the other hand, polyploids are frequently linked with distinct traits such as “gigas effects”, which describes increased size and plant organs.
Additionally, benefits of polyploidy can include enhanced capabilities for buffering of deleterious mutation (due to increased heterozygosity), hybrid vigour (heterosis), and–somewhat contentiously–reversal of selfing inhibition <span class="citation">(Woodhouse, Burkart-Waco, and Comai <a href="#ref-Woodhouse2009">2009</a>; Ramsey and Ramsey <a href="#ref-Ramsey2014">2014</a>)</span>.
These traits are thought to overcome the reproductive disadvantages of polyploidy and, in those rare cases, make this mutation key to the invasive and adaptive potential of plants. Let’s examine these mechanisms in more detail:</p>
</div>
<div id="benefits" class="section level2">
<h2><span class="header-section-number">1.2</span> Benefits</h2>
<div id="gigas" class="section level3">
<h3><span class="header-section-number">1.2.1</span> Gigas-effects</h3>
</div>
<div id="buffering" class="section level3">
<h3><span class="header-section-number">1.2.2</span> Genetic buffering</h3>
</div>
<div id="vigour" class="section level3">
<h3><span class="header-section-number">1.2.3</span> Hybrid Vigour</h3>
</div>
</div>
<div id="costs" class="section level2">
<h2><span class="header-section-number">1.3</span> Costs</h2>
<blockquote>
<p>Are they realistic? How do they work? Why/when are they important? consider the conditions.</p>
</blockquote>
<div id="pollen-swamping" class="section level3">
<h3><span class="header-section-number">1.3.1</span> Diploid Pollen-Swamping</h3>
</div>
<div id="flowering" class="section level3">
<h3><span class="header-section-number">1.3.2</span> Delayed Flowering</h3>
</div>
<div id="sterility" class="section level3">
<h3><span class="header-section-number">1.3.3</span> Triploid Sterility</h3>
<blockquote>
<p>So what are the core mechanisms?</p>
</blockquote>
<blockquote>
<p>Or the most suspicious?</p>
</blockquote>
</div>
</div>
<div id="how-do-these-mechanisms-link" class="section level2">
<h2><span class="header-section-number">1.4</span> How do these mechanisms link?</h2>
<blockquote>
<p>Does limited mate-choice set the scene for the evolution of selfing vs out-crossing?</p>
</blockquote>
<blockquote>
<p>Delayed flowering (cost) is associated with gigas-effects of increased size (benefit). TRADE-OFF.</p>
</blockquote>
</div>
<div id="aims" class="section level2">
<h2><span class="header-section-number">1.5</span> Aims</h2>
<p>Based on all this pondering - what am I actually going to examine?</p>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-Arrigo2012">
<p>Arrigo, Nils, and Michael S Barker. 2012. “Rarely successful polyploids and their legacy in plant genomes.” <em>Current Opinion in Plant Biology</em> 15 (2): 140–46. <a href="https://doi.org/10.1016/j.pbi.2012.03.010">https://doi.org/10.1016/j.pbi.2012.03.010</a>.</p>
</div>
<div id="ref-Jiao2011">
<p>Jiao, Yuannian, Norman J. Wickett, Saravanaraj Ayyampalayam, André S. Chanderbali, Lena Landherr, Paula E. Ralph, Lynn P. Tomsho, et al. 2011. “Ancestral polyploidy in seed plants and angiosperms.” <em>Nature</em> 473 (7345): 97–100. <a href="https://doi.org/10.1038/nature09916">https://doi.org/10.1038/nature09916</a>.</p>
</div>
<div id="ref-Mayrose2011">
<p>Mayrose, Itay, Shing H. Zhan, Carl J. Rothfels, Karen Magnuson-Ford, Michael S. Barker, Loren H. Rieseberg, and Sarah P. Otto. 2011. “Recently Formed Polyploid Plants Diversify at Lower Rates.” <em>Science</em> 333 (6047): 1257. <a href="https://doi.org/10.1126/science.1207205">https://doi.org/10.1126/science.1207205</a>.</p>
</div>
<div id="ref-Ramsey2014">
<p>Ramsey, Justin, and Tara S Ramsey. 2014. “Ecological studies of polyploidy in the 100 years following its discovery.” <em>Phil. Trans. R. Soc. B</em> 369. <a href="https://doi.org/10.1098/rstb.2013.0352">https://doi.org/10.1098/rstb.2013.0352</a>.</p>
</div>
<div id="ref-Wood2009">
<p>Wood, Troy E, Naoki Takebayashi, Michael S Barker, Itay Mayrose, Philip B Greenspoon, and Loren H Rieseberg. 2009. “The frequency of polyploid speciation in vascular plants.” <em>Proceedings of the National Academy of Sciences of the United States of America</em> 106 (33). National Academy of Sciences: 13875–9. <a href="https://doi.org/10.1073/pnas.0811575106">https://doi.org/10.1073/pnas.0811575106</a>.</p>
</div>
<div id="ref-Woodhouse2009">
<p>Woodhouse, M., D. Burkart-Waco, and L. Comai. 2009. “Polyploidy.” <em>Nature Education</em> 2 (1): 1. <a href="https://www.nature.com/scitable/topicpage/polyploidy-1552814">https://www.nature.com/scitable/topicpage/polyploidy-1552814</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>

<a href="references.html" class="navigation navigation-next navigation-unique" aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/rstudio/bookdown-demo/edit/master/intro.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"download": ["thesis_2417024.pdf", "thesis_2417024.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
